src/com/t_oster/liblasercut/drivers/GenericGcodeDriver.java

```diff
@@ -523,7 +523,7 @@ private void writeShutdownCode() throws IOException {
 
   protected void sendLine(String text, Object... parameters) throws IOException
   {
-    out.format(FORMAT_LOCALE, text+LINEEND(), parameters);
+    out.format(FORMAT_LOCALE, text.replace(" ", "")+LINEEND(), parameters);
```

Comment:
would you mind overriding this method in the GRBL driver, because I am not sure if other G-Code interpreters need the spaces...